docs/docs/icicle/primitives/hash.md

```diff
@@ -71,6 +71,83 @@ The S box power alpha, number of full rounds and partial rounds, rounds constant
 
 In the current version the padding is not supported and should be performed by the user.
 
+## Supported Bindings
```

Comment:
Why is this section Poseidon2 specific?
If we want to keep it poseidon2 specific, it should be a header 4 under the Poseidon2 header 3.